tests/std/tests/P0896R4_views_split/test.cpp

```diff
@@ -320,7 +321,32 @@ constexpr bool instantiation_test() {
     return true;
 }
 
+constexpr bool test_1559808() {
+    // Regression test for DevCom-1559808, an interaction between vector and the
+    // use of structured bindings in the constexpr evaluator.
+
+    vector<char> letters{'T', 'h', 'i', 's', ' ', 'i', 's', ' ', 'a', ' ', 't', 'e', 's', 't'};
+    auto r = views::split(letters, ' ');
+    auto i = r.begin();
+    assert(*(*i).begin() == 'T');
+    ++i;
+    assert(*(*i).begin() == 'i');
+    ++i;
+    assert(*(*i).begin() == 'a');
+    ++i;
+    assert(*(*i).begin() == 't');
+    ++i;
+    assert(i == r.end());
+
+    return true;
+}
+
 int main() {
     STATIC_ASSERT(instantiation_test());
     instantiation_test();
+
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, DevDiv-1516290
+    STATIC_ASSERT(test_1559808());
+#endif // TRANSITION, DevDiv-1516290

```

Comment:
It appears that this intends to cite DevCom-1516290 "C++20 constexpr dynamic allocation doesn't properly understand std::construct_at()", not the much older DevDiv internal database.